varia/sass/components/comments/_comments.scss

```diff
@@ -144,7 +144,7 @@
 			.avatar {
 				margin-right: #{map-deep-get($config-global, "spacing", "horizontal")};
 				display: inherit;
-				position: inherit;
+				position: relative;
```

Comment:
I think the `position` can be just removed entirely, no?

Reply:
Nevermind. I see that otherwise it gets `position:absolute`, so that has to be overridden.

It's important for `amp-img` to get `position:relative` (or `position:absolute`) because the nested `img` element needs the parent `amp-img` to have a new stacking context so that it can fill its parent when the `img` does `position:absolute; top:0; right:0; bottom:0; left:0;`. 

The `amp-img` has `position:relative` by default for images (with intrinsic layout, the default as output by the AMP plugin): 

https://github.com/ampproject/amphtml/blob/e1a66bce763efff45522ab77e15cdc7ff2aaebc7/css/ampshared.css#L78-L84

So that `position` needs to not be overridden with a value that removes the new CSS stacking context.

Reply:
Removing position entirely somehow creates different issue. Somehow it takes absolute position and hides the author name behind the avatar. Having relative position seems to be fixing this theme issue for AMP. 
 
![Screenshot 2020-05-14 at 11 54 28 PM](https://user-images.githubusercontent.com/1139853/81971865-646a1000-963f-11ea-90c9-a9a58d4763ef.png)